lib/internal/webstreams/readablestream.js

```diff
@@ -642,26 +641,37 @@ ObjectDefineProperties(ReadableStream.prototype, {
   [SymbolToStringTag]: getNonWritablePropertyDescriptor(ReadableStream.name),
 });
 
-function TransferredReadableStream() {
-  return ReflectConstruct(
-    function() {
-      markTransferMode(this, false, true);
-      this[kType] = 'ReadableStream';
-      this[kState] = {
-        disturbed: false,
-        state: 'readable',
-        storedError: undefined,
-        stream: undefined,
-        transfer: {
-          writable: undefined,
-          port: undefined,
-          promise: undefined,
-        },
-      };
-      this[kIsClosedPromise] = createDeferredPromise();
+function InternalTransferredReadableStream() {
+  markTransferMode(this, false, true);
+  this[kType] = 'ReadableStream';
+  this[kState] = {
+    disturbed: false,
+    reader: undefined,
+    state: 'readable',
+    storedError: undefined,
+    stream: undefined,
+    transfer: {
+      writable: undefined,
+      port1: undefined,
+      port2: undefined,
+      promise: undefined,
```

Comment:
For these properties, I followed what I saw in the constructor, the old version was wrong, right?